### DISCUSSION TITLE
Null record on custom page action

### DISCUSSION THREAD

**Discussion by ramiropiro** [filamentphp/filament#5311] [General]
I'm tring to get the value from a list on a custom page, so i can trigger an action to create a pdf, while i'm getting the correct record on the blade file and genereting the view fine, i'm not able to get the record on the custom page.

![image](https://user-images.githubusercontent.com/111913304/209016828-f7fadb9f-1078-4bf1-a064-9aee5cf3905f.png)

![image](https://user-images.githubusercontent.com/111913304/209016741-ffdd993d-429f-4207-a29e-ffe773d5be14.png)

**Comment by ArtMin96** [comment]
Try this

`public ?RemuneracionHora $record = null;` to `public ?RemuneracionHora $record;`

And if this doesn't work add this method

```php
public static function getSlug(): string
{
    return 'some-url/{record}'; // <- The record must have the same name as the property
}
```
